modules/git/command.go

```diff
@@ -104,6 +106,16 @@ type RunOpts struct {
 	PipelineFunc   func(context.Context, context.CancelFunc) error
 }
 
+func CommonEnvs() []string {
+	return []string{
+		fmt.Sprintf("LC_ALL=%s", DefaultLocale),
+		"GIT_TERMINAL_PROMPT=0",    // avoid prompting for credentials interactively, supported since git v2.3
+		"GIT_NO_REPLACE_OBJECTS=1", // ignore replace references (https://git-scm.com/docs/git-replace)
+		"GIT_CONFIG_NOSYSTEM=1",    // https://git-scm.com/docs/git-config, and GIT_CONFIG_GLOBAL, they require git >= 2.32
+		"HOME=" + HomeDir,          // make Gitea use internal git config only, to prevent conflicts with user's git config
```

Comment:
The filename is specific for `HOME` directory.

Reply:
Sorry, what do you mean? I haven't got your point.